gen.yaml

```diff
@@ -34,7 +34,7 @@ typescript:
       shared: ""
       webhooks: ""
   installationURL: https://github.com/hathora/cloud-sdk-typescript
-  maxMethodParams: "3"
+  maxMethodParams: 3
```

Comment:
This was a bug in the gen.yaml that needs to be fixed to move to a later speakeasy version. maxMethodParams is an int not string.

Reply:
@ryan-timothy-albert cool just curious why 3? in our monorepo we have this set to 4. i don't particularly care

Reply:
Oh I just stuck with the value that's in main right now in this repo. 3 or 4 doesn't matter just has to be a number.

Reply:
kk